src/main/java/com/coveros/selenified/element/check/Constants.java

```diff
@@ -0,0 +1,59 @@
+package com.coveros.selenified.element.check;
+
+public class Constants {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1118)